.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
It is better to turn off trimming for `.md` files. It could cause unwanted behavior.

Reply:
```
[*.md]
trim_trailing_whitespace = false
```